mvn compile: missing dependency

```[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /home/rico/minecraft-json-telemetry/src/main/java/ch/heap/bukkit/epilog/Updater.java:[23,22] package javax.xml.bind does not exist
[ERROR] /home/rico/minecraft-json-telemetry/src/main/java/ch/heap/bukkit/epilog/Updater.java:[269,41] cannot find symbol
  symbol:   variable DatatypeConverter
  location: class ch.heap.bukkit.epilog.Updater
[INFO] 2 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.657 s
[INFO] Finished at: 2023-07-31T21:49:27+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project Epilog: Compilation failure: Compilation failure:
[ERROR] /home/rico/minecraft-json-telemetry/src/main/java/ch/heap/bukkit/epilog/Updater.java:[23,22] package javax.xml.bind does not exist
[ERROR] /home/rico/minecraft-json-telemetry/src/main/java/ch/heap/bukkit/epilog/Updater.java:[269,41] cannot find symbol
[ERROR]   symbol:   variable DatatypeConverter
[ERROR]   location: class ch.heap.bukkit.epilog.Updater
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException```

--- a/README.md
+++ b/README.md
@@ -1,15 +1,17 @@
 # Minecraft Epilog plugin compatible with JSON server
 
-Telemetry plugin for bukkit compatible Minecraft servers.
-It send telemetry data provided by EventHandlers to the json server.
-Tested with version Minecraft server: 1.12.2
+Telemetry plugin for bukkit compatible Minecraft servers. It sends telemetry data provided by EventHandlers to the JSON Server. Tested with version Minecraft server: 1.12.2
 
-Json server in docker is compatible with the plugin is : [clue-json-server](https://github.com/clue/docker-json-server)
+JSON Server in Docker compatible with the plugin is : [clue-json-server](https://github.com/clue/docker-json-server)
 
-More information can be found at [heapcraft.net](http://heapcraft.net/?p=epilog-manual)
-More information can be found at [dmytro.shytyi.net](http://dmytro.shytyi.net)
-More information can be found at [Minecarft-elastic-kibana-visualization](https://dmytro.shytyi.net/data-visualization-from-my-minecraft-server-with-elasticsearch-and-kibana/)
+Java dependencies for Maven compiler are configured in Project Object Model (POM): [pom.xml](pom.xml).
+
+More information can be found at:
+* [heapcraft.net](http://heapcraft.net/?p=epilog-manual)
+* [dmytro.shytyi.net](http://dmytro.shytyi.net)
+* [Minecarft-elastic-kibana-visualization](https://dmytro.shytyi.net/data-visualization-from-my-minecraft-server-with-elasticsearch-and-kibana/)
 
 # Credits:
-stepmuel <stephan@heap.ch>
-dmytroshytyi <dmytro@shytyi.net>
+* stepmuel <stephan@heap.ch>
+* dmytroshytyi <dmytro@shytyi.net>
+

--- a/pom.xml
+++ b/pom.xml
@@ -28,5 +28,16 @@
 			<artifactId>bukkit</artifactId>
 			<version>1.12.1-R0.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>2.3.3</version>
+		</dependency>
+		<dependency>
+			<groupId>com.sun.xml.bind</groupId>
+			<artifactId>jaxb-impl</artifactId>
+			<version>2.3.3</version>
+			<scope>runtime</scope>
+		</dependency>
 	</dependencies>
 </project>

